### ISSUE TITLE
Daily Quest: Shredder Repair

### ISSUE THREAD

**Issue by Warlockboss** [dalaranwow/dalaran-wow#451]
http://www.wowhead.com/quest=12244
http://www.wowhead.com/npc=27354
It seems that you can't hop in to shredders so you can't deliver them to finish the quest.
video http://www.youtube.com/watch?v=7aVWto_rhGQ

**Comment by Cone182** [comment]
Well I could jump into one shredder, but the other ones were not mountable. Maybe it really works only once per restart....

**Comment by ashido96** [comment]
For horde side, this quest works if you turn the shredder in at the base of the ramp leading up into the cart next to the quest giver.

**Comment by ashido96** [comment]
As came back today to attempt the daily again, every shredder was uninteractable, couldn't even see the icon to get in. This quest is **NOT** fixed, and should be re-opened.

**Comment by Vaxsysl** [comment]
Still bugged, not intractable at all.

**Comment by cover25** [comment]
Can confirm, doesn't have an option to enter the shredder.

**Comment by Rushor** [comment]
the spell_click_spell gets removed after the vehicle respawns

adding the spell_klick_flag all the times in creature_template fixed this issue

**Comment by Omuno** [comment]
As of Today 27th July 2015 this quest can not be completed, there are 4 shredder in the area that i found, i manage to find one that i can click (i can't click the other three) and bring it successfully. But as it respawn again, i cannot ride it again. It seemed to me that the problem is that the shredder is delivered using an ability, we can get out of the shredder this way, but the shredder stuck with a state that someone was riding them (so we can't ride it when is spawn).

**Comment by sklivvz** [comment]
Issue is still there, shredders are not usable after respawn

**Comment by iamralph** [comment]
and now you too will be kicking yourself for even reporting it with endless emails about it *unsubs finally*

**Comment by NovaHiros** [comment]
Still bugged. Come on!

**Comment by Lothloryen** [comment]
Hello there,

Thank for the reports, it will be corrected after the next realm restart.

My apologies, totally missed this one and Rushor's suggestion easily fix this issue - thank you, Rushor.

**Comment by Rushor** [comment]
@Lothloryen: let me know if you want to get the battle for the undercity fight. i would really like to share much code with you <3

**Comment by Lothloryen** [comment]
Hello Rushor,

Thank you for the offer, I started working on The Battle for The Undercity some time ago, but unfortunately there were many things going on at the same time so I have it on hold. Anyways, if you wish, you can write me to: dalaran.community@gmail.com (I may be a bit delayed answering sometimes). Really appreciated your help.
